orchestrator/redisClientWrapper.js

```diff
@@ -0,0 +1,109 @@
+/* jslint node: true */
+"use strict";
+var axios = require("axios");
+
+var config = require('./config');
+var auth = require('./auth');
+
+/**
+ * Client for REDIS database
+ */
+class ClientWrapper {
+
+  constructor(client) {
+    this.client = client;
+  }
+  /**
+   * Returns template list of a device. Search for it on cache, if it's not, requests device manager
+   * and then stores on cache.
+   * @param {string} deviceid
+   * @param {string} tenant 
+   * @param {string} redisState
+   */
+
+  getTemplateList(deviceid, tenant, redisState) {
```

Comment:
It seems the parameter order: tenant, deviceid, redisState would be more logical.

Reply:
I guess that too.